db/init.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS User (
+    usr_id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(64) NOT NULL UNIQUE,
+    password VARCHAR(72) NOT NULL, -- bcryptの最大長が72らしい
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS Tweet (
+    tw_id BIGINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    usr_id INT NOT NULL,
+    body VARCHAR(280) NOT NULL,
+    replied_to INT NULL,
```

Comment:
```suggestion
    replied_to BIGINT NULL,
```
`tw_id`をBIGINTにする以上はここもBIGINTにした方が良きかな？

Reply:
BIGINT UNSIGNEDですね、直します